lit_gpt/tokenizer.py

```diff
@@ -61,17 +72,21 @@ def encode(
         self,
         string: str,
         device: Optional[torch.device] = None,
-        bos: bool = False,
```

Comment:
You could instead make this `Optional[bool]` so that passing a boolean is interpreted as overriding the default

Reply:
@carmocca  updated, marking as resolved